code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -29,6 +29,12 @@
 		return 0
 	.=..()
 
+/mob/living/carbon/human/experience_pressure_difference()
+	playsound(src, 'sound/effects/space_wind.ogg', 50, 1)
+	if(shoes.flags&NOSLIP)

```

Comment:
this means the janitor will be immune to spacewind---which makes no sense; shouldn't this check for MAGPULSE on magboots, specifically?